Framework/DataHandling/src/ProcessBankData.cpp

```diff
@@ -242,5 +242,20 @@ void ProcessBankData::run() { // override {
 #endif
 } // END-OF-RUN()
 
+// Get the workspace index for a given pixel ID. Throws if the pixel ID is
+// not in the expected range.
+size_t ProcessBankData::getWorkspaceIndexFromPixelID(const detid_t pixID) {
```

Comment:
You should add the '@param' and '@return' tags to the method description for parameters and return respectively.